demo/app/App_Resources/Android/app.gradle

```
@@ -6,11 +6,28 @@
 //}
 
 android {  
-  defaultConfig {  
+  defaultConfig {
+    applicationId = "nl.essent.urbanairship"
```

Is nl.essent.urbanairship supposed to be in there?
```suggestion
    applicationId = "org.nativescript.demo"
```

---

It is a demo app from NativeScript, we only added some code to use the plugin, that's why it's now the original app id.